Dockerfile.cli

```diff
@@ -8,9 +8,20 @@ RUN /root/scripts/install-composer.sh
 
 WORKDIR /srv/annotations
 COPY composer.json composer.lock /srv/annotations/
+# could probably use --no-dev and other optimizations,
+# if we specialize annotations_ci instead
 RUN composer install
 
-COPY . /srv/annotations
+# TODO: remove phpunit.xml.dist from here?
+# yes this is how you copy directories
```

Comment:
alternative is to `COPY . /srv/annotations` and relying on .dockerignore, but it cannot exclude files that only need to be sent to one image of the 3 like `smoke_tests.sh`

Reply:
`rm`?

Reply:
What happens if you copy an ignored file?

Reply:
the build phase is executed by the Docker daemon, which is not sent the ignored files, meaning the file is not there to be copied